docs/proposals/20220603-kubernetes-vendor.md

```diff
@@ -0,0 +1,297 @@
+# Multi-runtime Support
+
+Authors:
+- [Ce Gao](https://github.com/gaocegege)
+
+## Summary
+
+envd builds and runs the environments with Docker. This proposal is to support Kubernetes as a runtime option. After this feature, envd can build the environment with Docker and run on Kubernetes.
```

Comment:
as https://github.com/tensorchord/envd/pull/303/files#diff-6d1dcf3ec1594cadeb70c98ab5a7c1864e15d436c11fedda2f95fdc083be9293R19 specified, envd can build envs with the buildkitd pod, so envd can both build envs and run them on kubernetes, right?

Reply:
Yep, these two features are de-coupled. Buildkit in the pod is already supported.